actions/lib/ansible_base.py

```diff
@@ -43,10 +43,13 @@ def _parse_extra_vars(self):
                 var_list = [n.strip() for n in var_list]
```

Comment:
This line will need to change because it assumes that ast.literal_eval will provide strings. However, I gave it a json object / dict.